docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -2503,16 +2503,24 @@ end
 | Yes
 | No
 | 1.0
-| -
+| 2.4
 |===
 
-Checks for multiple top-level example groups.
+Checks for multiple top-level example groups or nested example groups.
+
+This cop can be configured using the `EnforcedStyle` option.
+
+Using the `EnforcedStyle: nesting`, multiple descriptions for the
```

Comment:
"multiple descriptions for the same class" - I believe this is a bit misleading. We're talking about example groups structure, and adjusting their descriptions (or titles, or docstrings) are just a side effect.

Reply:
Yes, that's true.
I corrected the sentence to use `example group` instead of the `multiple descriptions`.